Jenkinsfile.sles15ncn

```diff
@@ -1,39 +0,0 @@
-/*
```

Comment:
Why don't we need this file any more?

Reply:
Mitch confirmed that this was Jenkins cruft that's not used since our move to github.